src/assets/TokenDetectionController.ts

```diff
@@ -137,6 +179,14 @@ export class TokenDetectionController extends BaseController<
     this.stopPolling();
   }
 
+  /**
+   * Restart polling for the token list.
+   */
+  async restart() {
```

Comment:
Can this be a private method? I don't think we need to call this from anywhere else. We only use `start` and `stop` externally.

Reply:
Was able to remove this method with a bit of refactor clean up